pkg/eigenState/operatorDirectedRewardSubmissions/operatorDirectedRewardSubmissions.go

```diff
@@ -0,0 +1,334 @@
+	arguments, err := odrs.ParseLogArguments(log)
+	if err != nil {
+		return nil, err
+	}
+
+	outputData, err := parseRewardSubmissionOutputData(log.OutputData)
+	if err != nil {
+		return nil, err
+	}
+	outputRewardData := outputData.OperatorDirectedRewardsSubmission
+
+	rewardSubmissions := make([]*OperatorDirectedRewardSubmission, 0)
+
+	for i, strategyAndMultiplier := range outputRewardData.StrategiesAndMultipliers {
+		startTimestamp := time.Unix(int64(outputRewardData.StartTimestamp), 0)
+		endTimestamp := startTimestamp.Add(time.Duration(outputRewardData.Duration) * time.Second)
+
+		multiplierBig, success := numbers.NewBig257().SetString(strategyAndMultiplier.Multiplier.String(), 10)
+		if !success {
+			return nil, xerrors.Errorf("Failed to parse multiplier to Big257: %s", strategyAndMultiplier.Multiplier.String())
+		}
+
+		for j, operatorReward := range outputRewardData.OperatorRewards {
+			amountBig, success := numbers.NewBig257().SetString(operatorReward.Amount.String(), 10)
+			if !success {
+				return nil, xerrors.Errorf("Failed to parse amount to Big257: %s", operatorReward.Amount.String())
+			}
+
+			rewardSubmission := &OperatorDirectedRewardSubmission{
+				Avs:             strings.ToLower(arguments[1].Value.(string)),
+				RewardHash:      strings.ToLower(arguments[2].Value.(string)),
+				Token:           strings.ToLower(outputRewardData.Token),
+				Operator:        strings.ToLower(operatorReward.Operator),
+				OperatorIndex:   uint64(j),
+				Amount:          amountBig.String(),
+				Strategy:        strings.ToLower(strategyAndMultiplier.Strategy),
+				StrategyIndex:   uint64(i),
+				Multiplier:      multiplierBig.String(),
+				StartTimestamp:  &startTimestamp,
+				EndTimestamp:    &endTimestamp,
+				Duration:        outputRewardData.Duration,
+				BlockNumber:     log.BlockNumber,
+				TransactionHash: log.TransactionHash,
+				LogIndex:        log.LogIndex,
+			}
+
+			rewardSubmissions = append(rewardSubmissions, rewardSubmission)
+		}
+	}
+
+	return rewardSubmissions, nil
+}
+
+func (odrs *OperatorDirectedRewardSubmissionsModel) GetStateTransitions() (types.StateTransitions[[]*OperatorDirectedRewardSubmission], []uint64) {
+	stateChanges := make(types.StateTransitions[[]*OperatorDirectedRewardSubmission])
+
+	stateChanges[0] = func(log *storage.TransactionLog) ([]*OperatorDirectedRewardSubmission, error) {
+		rewardSubmissions, err := odrs.handleOperatorDirectedRewardSubmissionCreatedEvent(log)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, rewardSubmission := range rewardSubmissions {
+			slotId := NewSlotID(rewardSubmission.TransactionHash, rewardSubmission.LogIndex, rewardSubmission.RewardHash, rewardSubmission.StrategyIndex, rewardSubmission.OperatorIndex)
+
+			_, ok := odrs.stateAccumulator[log.BlockNumber][slotId]
+			if ok {
+				fmt.Printf("Submissions: %+v\n", odrs.stateAccumulator[log.BlockNumber])
```

Comment:
Make sure to remove this

Reply:
Done in [1c3161b](https://github.com/Layr-Labs/sidecar/pull/112/commits/1c3161b691271125c3c7f669d3a47b6e015f1e9c)